bloom-housing/bloom
# feat: geocoding preference set up fixed layers

# Pull Request Template

## Issue Overview

This PR addresses #3783


## Description

Creates new backend table for mapLayers, and adds additional field for multiselect-option (to store selected map id).
On frontend adds select within geocoding section in settings / preferences -> options.
There is discrepancy with design (removed link "contact us"), it was consulted with Kathy as a way to go for now.

## How Can This Be Tested/Reviewed?

Run new migration, or reseed db.
in partners on `/settings` pick preference, here you can pick jurisdiction to restrict map layers in options select.
go to option then `Do you want to collect address information?` -> yes , `Yes, check with ArcGIS map` it will show new select. Now test if that saves correctly etc.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
